android/capacitor/src/main/java/com/getcapacitor/Bridge.java

```diff
@@ -765,42 +763,29 @@ public void startActivityForPluginWithResult(PluginCall call, Intent intent, int
      * @param requestCode the code that was requested
      * @param permissions the permissions requested
      * @param grantResults the set of granted/denied permissions
+     * @return true if permission code was handled by a plugin explicitly, false if not
```

Comment:
I think we should somehow indicate in the javadoc that this method is for Cordova plugins and legacy plugins only. It may be a good place to document how all the new permission handling works, because it's complex. Either here or `BridgeActivity.java`, what do you think?